js/atree.js

```diff
@@ -1172,6 +1136,21 @@ function atree_set_state(node_wrapper, new_state) {
     }
 };
 
+// refresh all connector to default state, then try to calculate the connector for all node
+function atree_update_connector() {
+    atree_connectors_map.forEach((v) => {
+        if (v.length != 0) {
+            let connector_elem = document.createElement("img");
+            connector_elem.style = "width: 100%; height: 100%;";
+            connector_elem.src = '../media/atree/connect_' + v[0].type + '.png'
+            v[0].replaceChildren(connector_elem);
+        }
+    });
+    atree_map.forEach((v) => {
+        atree_compute_highlight(v);
+    });
+}
+
```

Comment:
i believe this is dead/unused code unless you are calling it somewhere

Reply:
It seems that this function was removed by another contributer and I missed it in merging